### DISCUSSION TITLE
请问一下封面，摘要，目录后都有空白页是怎么回事？

### DISCUSSION THREAD

**Discussion by CNXZY** [note286/xduts#115] [Q&A] [answered]
目前来看是前言的偶数页这样，如何去除多余的空白页呢？本人初学者，恳请大佬赐教！

**Comment by note286** [comment, marked as answer]
论文需要打印；无法去除。

**Reply by CNXZY** [reply]
那么有没有办法改变前言的罗马数字页码呢

**Reply by sikouhjw** [reply]
> 那么有没有办法改变前言的罗马数字页码呢

有没有可能是学校规范要求为罗马数字呢？

**Reply by CNXZY** [reply]
哦我的意思是，在pdf软件里删了空白页，删完后页码还是正常顺序。

**Reply by note286** [reply]
你考虑过页眉和双面打印吗？

**Reply by sikouhjw** [reply]
> 哦我的意思是，在pdf软件里删了空白页，删完后页码还是正常顺序。

你是听不懂回答吗，说的很清楚，学校要求有空白页，为什么要删除？

**Reply by CNXZY** [reply]
> 你考虑过页眉和双面打印吗？

原来如此，谢谢
